deploy/base/run-maven-component-build-v0.1.yaml

```diff
@@ -84,22 +84,33 @@ spec:
   steps:
     - name: git-clone
       image: "gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init:v0.21.0"
+      resources:
+        requests:
+          memory: "128Mi"
+          cpu: "10m"
+        limits:
+          memory: "512Mi"
```

Comment:
Pretty sure this will fail for Quarkus. I will follow up and make this be parameterized so it can be overridden for large builds.

Reply:
this on git-clone, mvn-goals is not modified